src/pdfs/abstract_cgaussian.jl

```diff
@@ -13,7 +13,8 @@ end
 
 function loglikelihood(p::AbstractCGaussian{T}, x::AbstractArray, z::AbstractArray) where T
     (μ, σ2) = mean_var(p, z)
-    y = (x - μ).^2
+    d = x - μ
+    y = d .* d
```

Comment:
Is this really an issue for everyone or can this be related to some old CUDA version or so?
`.^2` complains for me that scalar getindex is disallowed